Dockerfile

```diff
@@ -1,24 +1,21 @@
-ARG ANOL_COMMIT_HASH=08b57ecd563a1add5b4bdacdd026ffe4df603d20
+ARG ANOL_COMMIT_HASH=63de4627b24c5b886352858c2e52f60dcc3f16c0
 
-FROM node:14.19.3-alpine3.14 AS clientbuilder
+FROM node:22-alpine@sha256:6e80991f69cc7722c561e5d14d5e72ab47c0d6b6cfb3ae50fb9cf9a7b30fdf97 AS clientbuilder
```

Comment:
Is there a reason for using a specific hash here?

Reply:
No, this is just for pinning the image. We have renovate installed which should inform us about newer hashes.